FAQ & Doc updates for stake delegation and mainnet worklock (#2195)
nucypher/nucypher: docs/source/architecture/worklock.rst

Overview
--------

> `WorkLock` is a novel, permissionless decentralized network bootstrapping mechanism developed at NuCypher which requires

Reviewer: ```suggestion
`WorkLock` is a novel, permissionless decentralized network node setup mechanism, developed at NuCypher, which requires
```

> participants to temporarily escrow ETH and operate NuCypher nodes in order to be allocated NU, the native

Reviewer 2: "_novel, permissionless decentralized network node setup mechanism_" <- That's a tough sentence to read, particularly if you're not native. It has 5 adjectives in a row!

> participants to temporarily escrow ETH and operate NuCypher nodes in order to be allocated NU tokens, the native

Reviewer: ```suggestion
participants to temporarily escrow ETH and operate NuCypher nodes in order to be allocated NU, the native
```

> token of the NuCypher network that enables node operation.

Reviewer 2: ```suggestion
cryptocurrency of the NuCypher network that enables node operation.
```

> WorkLock is designed onboard threshold cryptography nodes to the live NuCypher network using a process that selects

Reviewer 3: ```suggestion
WorkLock is designed to onboard threshold cryptography nodes to the live NuCypher network using a process that selects
```

> participants who are most likely to strengthen the network by committing to staking and running active nodes.

Reviewer 3: ```suggestion
participants who are most likely to strengthen and sustain the network by committing to staking and running active nodes.
```

> The WorkLock begins with a contribution period, during which anyone seeking to operate a nucypher node can send

Reviewer 4: ```suggestion
The WorkLock begins with a contribution period, during which anyone seeking to operate a NuCypher node can send
```

> The WorkLock begins with a contribution period, during which anyone seeking to operate a NuCypher node can send

Reviewer: ```suggestion
The WorkLock begins with an escrow period, during which anyone seeking to operate a NuCypher node can send
```

ETH to the WorkLock contract to be temporarily escrowed on-chain.

> At any time during the contribution period, WorkLock participants can cancel their participation to forgo NU and recoup

Reviewer: ```suggestion
At any time during the escrow period, WorkLock participants can cancel their participation to forgo NU and recoup
```

their escrowed ETH immediately.

> Once the contribution period closes, the WorkLock contract does not accept more ETH, but will still accept cancellations

Reviewer: ```suggestion
Once the escrow period closes, the WorkLock contract does not accept more ETH, but will still accept cancellations
```

during an additional time window (the "cancellation window").  At the end of this cancellation period,

> the token allocation period begins and stake-locked NU are allocated to participants in the form of non-transferable stakes.

Reviewer: ```suggestion
the allocation period begins and stake-locked NU are allocated to participants in the form of non-transferable stakes.
```
